OpenRA.Mods.Common/Traits/Crushable.cs

```diff
@@ -80,14 +88,14 @@ bool CrushableInner(BitSet<CrushClass> crushClasses, Player crushOwner)
 			return Info.CrushClasses.Overlaps(crushClasses);
 		}
 
-		bool ICrushable.TryCalculatePlayerBlocking(Actor self, BitSet<CrushClass> crushClasses, out LongBitSet<PlayerBitMask> blocking)
+		protected override void TraitEnabled(Actor self)
 		{
-			if (IsTraitDisabled || !self.IsAtGroundLevel() || !Info.CrushClasses.Overlaps(crushClasses))
-				blocking = self.World.AllPlayerMask;
-			else
-				blocking = Info.CrushedByFriendlies ? default(LongBitSet<PlayerBitMask>) : self.Owner.AllyMask;
+			self.World.ActorMap.MarkAsDirty(self.OccupiesSpace);
```

Comment:
Can we achieve the same thing by calling `ActorMap.UpdatePosition(self.OccupiesSpace)` instead?